### PR TITLE
[DDW-880] Forward data attributes to input field components

### PR DESCRIPTION
**Pull request by renanvalentin**
This PR forwards html attributes to input field to be used by integration tests.

Added `htmlFor` attribute to `label` node to create a relationship between label and form element id. 
```js
<label for="username-input">Username</label>
<input id="username-input" />

import {screen} from '@testing-library/react'
const inputNode = screen.getByLabelText('Username')
```

Support `data-*` and `aria-*` attributes as well:
```js
<div data-testid="custom-element" />

import {screen} from '@testing-library/dom'
const element = screen.getByTestId('custom-element')
```

### CODE REVIEW

**Comment by mchappell** [comment]
This will need a version bump for react-polymorph, have you been added to the npm package as an author?

**Review by lucas-barros — DISMISSED** [review]

**Review by DominikGuzei — DISMISSED** [review]
Looks great 👍

File: .husky/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+_
```
Comment (mchappell) [inline .husky/.gitignore:1]: why `-`?

File: package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-polymorph",
   "description": "React components with highly customizable logic, markup and styles.",
-  "version": "1.0.1",
+  "version": "1.0.1-next.3",
```
Comment (mchappell) [inline package.json:4]: once this pre-release has been tested, please publish a new version to NPM, and update the version to no longer point to next, but `1.0.2`

**Review by mchappell — CHANGES_REQUESTED** [review]

File: .husky/.gitignore (hunk 1 above)
Comment (renanvalentin) [reply .husky/.gitignore:1]: Dunno. Maybe husky has added it automatically. Should we remove this?

**Review by gabriela-ponce — APPROVED** [review]

**Review by miorsufianiohk — DISMISSED** [review]
LGTM. Great work @renanvalentin . Tested on Storybook 👍

File: .husky/.gitignore (hunk 1 above)
Comment (mchappell) [reply .husky/.gitignore:1]: From looking at the rest of the files, looks like husky uses that folder location to run the pre-commit script so lets keep.

**Review by mchappell — APPROVED** [review]

**Pull request merged**